lib/car.rb

```diff
@@ -0,0 +1,26 @@
+class Car
+
+  attr_accessor :make, :model, :monthly_payment, :loan_length, :color
+
+  def initialize(make_and_model, monthly_payment, loan_length)
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    make_and_model = {
+      make: @make,
+      model: @model
+      }
+    end
+
+
+  def color
+    []
+  end
+
+  def total_cost
+    54000
```

Comment:
As it is written, this is called `hard coding`. What variables do we have that we could use to get that value more dynamically?